server/src/OnDeviceComponent.ts

```diff
@@ -54,7 +48,7 @@ export class OnDeviceComponent {
 		this.conditionallyAddDefaultNodeReferenceKey(args);
 
 		const result = await this.sendRequest('callFunc', args, options);
-		return result.body as {
+		return result.json as {
 			value: any
 		} & ODC.ReturnTimeTaken;
```

Comment:
For all of these `& ODC.ReturnTimeTaken` parts, would it make more sense to actually define the types to include that information? That way external consumers can have an actual type to reference. Like this:
```
export interface CallfuncResponse extends ODC.ReturnTimeTaken {
    value: any;
}

...
return result.json as CallfuncResponse;
```

Reply:
There is only input arg types. Each function is the only spot where these are defined at.